src/lib/y2partitioner/widgets/pages/system.rb

```diff
@@ -96,18 +98,24 @@ def header
         def table
           return @table unless @table.nil?
 
-          @table = ConfigurableBlkDevicesTable.new(devices, @pager)
+          @table = ConfigurableBlkDevicesTable.new(devices, @pager, device_buttons)
           @table.remove_columns(:start, :end)
           @table
         end
 
+        # Widget with the dynamic set of buttons for the selected row
+        #
+        # @return [DeviceButtonsSet]
+        def device_buttons
+          @device_buttons ||= DeviceButtonsSet.new(pager)
+        end
+
         # Page buttons
         #
         # @return [Array<Yast::UI::Term>]
         def buttons
           buttons = [rescan_devices_button]
           buttons << import_mount_points_button if Yast::Mode.installation
-          buttons << HStretch()
```

Comment:
is there some agreement or decision to change how buttons look like? I see screenshot and not 100% sure it is better as it can have some original intention.

Reply:
I was discussing it for a while with @ancorgs locally. I proposed it (to have the discussion locally) in the daily call this morning, to make it faster and avoid to upload a bunch of useless screenshots.

Reply:
When we revamped the Partitioner interface in 15.1 we adopted a consistent buttons layout (agreed after many conversations among UX experts and developers) in all screens containing tables of devices except in two:

  - The NFS one because it's rendered by yast2-nfs-client
  - This one... for no particular reason. We simply did run out of time and energy to discuss it.

Since then, we have been considering the reasons for the old layout (two buttons at the left and another one at the right)... and we have found none. I guess it simply looked nicer to someone.

So we decided the time has come to make this screen consistent with all the others (well, except NFS, but that's another story).

And that consistency implies:
 - The first line of buttons is "contextual" (depends on the selected item) and aligned to left
 - The second line are global buttons (they affect the whole list) and are aligned to right